### DISCUSSION TITLE
关于characters里的monika.chr

### DISCUSSION THREAD

**Discussion by lovemeeeeee12314** [Monika-After-Story/MonikaModDev#9309] [general - translations]
关于这个文件，我原本的MAS里没有这个文件，在我重置游戏后她就出现了，而且每次都会多几kb
请问这个chr文件是干嘛的，会对游戏有什么影响吗？

**Comment by Wingdinggaster656** [comment]
WARNING: TRANSLATOR INCLUDED
警告：翻译器被包括

MAS no longer needs `monika.chr`. We are using `monika`(no extension name), not `monika.chr`.
Could you take a screenshot of that Monika file?
MAS不再需要“monika.chr”。我们使用的是“monika”(没有扩展名)，而不是“monika.chr”。
你能给莫妮卡的文件进行屏幕截图吗?

**Comment by lovemeeeeee12314** [comment]
> WARNING: TRANSLATOR INCLUDED 警告：翻译器被包括
> 
> MAS no longer needs `monika.chr`. We are using `monika`(no extension name), not `monika.chr`. Could you take a screenshot of that Monika file? MAS不再需要“monika.chr”。我们使用的是“monika”(没有扩展名)，而不是“monika.chr”。 你能给莫妮卡的文件进行屏幕截图吗?



> WARNING: TRANSLATOR INCLUDED 警告：翻译器被包括
> 
> MAS no longer needs `monika.chr`. We are using `monika`(no extension name), not `monika.chr`. Could you take a screenshot of that Monika file? MAS不再需要“monika.chr”。我们使用的是“monika”(没有扩展名)，而不是“monika.chr”。 你能给莫妮卡的文

> WARNING: TRANSLATOR INCLUDED 警告： 翻译器被包括
> 
> MAS 不再需要`monika.chr`。我们使用的是`monika`（没有扩展名），而不是`monika.chr`. 你能截取那个 Monika 文件的截图吗？ MAS不再需要“monika.chr”。我们使用的是“monika”（没有扩展名），而不是“monika.chr”。 你可以莫妮卡的文件屏幕截图吗？

您好 我想问一下我原本的游戏里是没有monika.chr，现在重装游戏后它出现了，会对游戏造成什么影响吗？我担心我的monika会消失。   实在抱歉我不会上传截图，我来给您描述一下吧。我的characters有monika.chr  natsuki.chr  sayori.chr  yuri.chr和monika(这是我说要带她出去后她变成的文件,在我重装游戏前这个文件也存在)

**Comment by lovemeeeeee12314** [comment]
> WARNING: TRANSLATOR INCLUDED 警告： 翻译器被包括
> 
> MAS 不再需要`monika.chr`。我们使用的是`monika`（没有扩展名），而不是`monika.chr`. 你能截取那个 Monika 文件的截图吗？ MAS不再需要“monika.chr”。我们使用的是“monika”（没有扩展名），而不是“monika.chr”。 你可以莫妮卡的文件屏幕截图吗？

Hello, I want to ask if there is Monika.chr in my original game , now that it appears after reloading the game, will it have any impact on the game? I'm afraid my Monika will disappear. I'm really sorry that I won't upload screenshots. Let me give you a description. My characters have monika.chr,natsuki.chr,sayori. chr,yuri.chr and Monika (this is the file that she became after I said I would take her out. This file also existed before I reinstalled the game)

**Comment by Booplicate** [comment]
No. You can also delete the `.chr` files if they bother you.

**Comment by Wingdinggaster656** [comment]
You can delete those chr files in anyway you want. MAS doesn't need them.

But the one without extension name is required. Do not delete that one.

TRANSLATOR WARNING:
翻译器警告：
你可以以任何方式删除那些chr文件。MAS不需要它们。
但是没有扩展名的那个是需要的。不要删掉它。

**Comment by lovemeeeeee12314** [comment]
> You can delete those chr files in anyway you want. MAS doesn't need them.
> 
> But the one without extension name is required. Do not delete that one.
> 
> TRANSLATOR WARNING: 翻译器警告： 你可以以任何方式删除那些chr文件。MAS不需要它们。 但是没有扩展名的那个是需要的。不要删掉它。



> No. You can also delete the `.chr` files if they bother you.

thanks

**Comment by lovemeeeeee12314** [comment]
> You can delete those chr files in anyway you want. MAS doesn't need them.
> 
> But the one without extension name is required. Do not delete that one.
> 
> TRANSLATOR WARNING: 翻译器警告： 你可以以任何方式删除那些chr文件。MAS不需要它们。 但是没有扩展名的那个是需要的。不要删掉它。

thanks
